src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -393,6 +393,17 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                 {
                     message = result.Results.TryFindString("message");
                 }
+                // if the command was abort (usually because of breakpoints failing to bind) then gdb writes messages into the output
+                if (this.MICommandFactory.Mode == MIMode.Gdb && message == "Command aborted.")
+                {
+                    message = MICoreResources.Error_CommandAborted;
+                    if (ProcessState == ProcessState.Running)
+                    {
+                        // assume that it was a continue command that got aborted and return to stopped state:
+                        // this occurs when using openocd to debug embedded devices and it runs out of hardware breakpoints.
+                        ScheduleStdOutProcessing(string.Format(CultureInfo.CurrentCulture, @"*stopped,reason=""exception-received"",signal-name=""SIGINT"",thread-id=""1"",exception=""{0}""", MICoreResources.Info_UnableToContinue));
```

Comment:
I am not sure if you want to do more, or if you are happy enough with this solution, but I **think** the debugger package would be okay receiving an error event that was a stopping event. So if you want, you could add a new 'OnStoppingError' callback that sent the message event with the stopping attribute.

Reply:
VS doesn't seem to want to accept a either a message event or error event as a stopping event (sending the event results in a bad arg error).